pyEpiabm/pyEpiabm/sweep/host_progression_sweep.py

```diff
@@ -26,6 +25,20 @@ def _update_time_to_status_change(self):
         new_time = random.randint(1, 10)
         return new_time
 
+    def _set_latent_time(self):
+        """Calculates and returns latency period as calculated in
+        covid-sim to be given as time until next infection status
+        for a person who has been set as exposed.
+        """
+        latent_period = pe.Parameters.instance().latent_period
+        latent_period_iCDF = pe.Parameters.instance().latent_period_iCDF
+        latent_icdf_object = InverseCdf(latent_period, latent_period_iCDF)
+        latent_time = latent_icdf_object.icdf_choose_exp()
+
+        if latent_time < 0:
+            raise AssertionError('Negative latent time')
+        return latent_time
```

Comment:
Do we want to handle a zero (or <1) latent time as well, or is this possible (and will this be set in the same time step?)

Reply:
Resolved by changing the call method in host progression. Instead of someone's status being updated if the current time is equal to their time of next status update, this has been changed to time of status up date less than or equal to the current time. This will handle the case of zero latent time.